README.md

```diff
@@ -156,6 +156,8 @@ The first two files will contain your App's ID, so take that in to consideration
 * Find the `Ovr Avatar (Script)` component within `+LocalAvatar`
 	* Ensure that `Show Third Person` is not checked.
 	* Optionally, un-check `Can Own Microphone` because we won't be using the mic in this tutorial.
+	
+* Create a manifest file, without it hands do not seem to work: Oculus -> Tools -> Create store-compatible AndroidManifest.xml
```

Comment:
What version of the Oculus Integration package did you test this with?  This is a requirement that was necessary between 1.36 and 1.39, but it went away in 1.40.  Are you running in to this today with the current 2019.2 version of Unity and Oculus Integration 12?  I definitely didn't need to do this step when I wrote the tutorial.

Could you rephrase this to be something like "If your hands aren't showing up, it may be necessary to create a Store Compatible AndroidManifest.xml file.  To do this, go to  `Oculus -> Tools -> Create store-compatible AndroidManifest.xml`"